sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ProgressReporter.scala

```diff
@@ -349,7 +349,7 @@ trait ProgressReporter extends Logging {
     result
   }
 
-  private def formatTimestamp(millis: Long): String = {
+  protected def formatTimestamp(millis: Long): String = {
     timestampFormat.format(new Date(millis))
```

Comment:
Not related to this PR, just want to confirm which timezone we expect.

The code here uses the JVM system timezone, while in Spark SQL we use the configured session local timezone.

cc @MaxGekk as well.

Reply:
This is also different from the previous code which use UTC: https://github.com/apache/spark/pull/27732/files#diff-3ff3668ba061d3b690c7bfe9e8f0db32L35